.eslintrc.json

```diff
@@ -1,8 +1,8 @@
 {
   "extends": [
+    "plugin:mattermost/react",
```

Comment:
The MM plugin should be last; if not their lint settings are not applied.

Reply:
not really.. having it last causes a million problems.

Reply:
can you point out why it should be last? btw it is first on v1

Reply:
Alright.  We can discuss it during our meetings.